composer.json

```diff
@@ -1,114 +1,113 @@
 {
```

Comment:
why was this entire file touched? spaces <-> tabs or something?
This will cause another big change during release.
Maybe have a look

Reply:
caused by the tagify update commit